<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset theme="moe.emmaexe.ntfyDesktop" resource="../../resources/resources.qrc">
    <normaloff>:/icons/ntfyDesktop.svg</normaloff>:/icons/ntfyDesktop.svg</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="buttons">
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="whatsThis">
        <string>This is a button that saves the configuration to the disk.</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset theme="document-save"/>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Notifications</string>
      </attribute>
      <layout class="QGridLayout" name="notifications">
       <item row="0" column="2">
        <widget class="QCheckBox" name="startupNotificationCheckbox">
         <property name="text">
          <string>Send the startup notification</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="keepHistoryLabel">
         <property name="text">
          <string>Keep</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QRadioButton" name="sourcePickerGlobal">
         <property name="text">
          <string>for a&amp;ll sources</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="3">
        <widget class="QComboBox" name="keepHistoryComboBox">
         <item>
          <property name="text">
           <string>all notifications</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a number of notifications</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>recent notifications</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>no notifications</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="errorNotificationCheckbox">
         <property name="text">
          <string>Send error notifications</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="aNumberLabel">
         <property name="text">
          <string>The number:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="2">
        <widget class="QComboBox" name="recentComboBox">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <item>
          <property name="text">
           <string>Seconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Months</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Years</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QRadioButton" name="sourcePickerIndividual">
         <property name="text">
          <string>&amp;for each source individually</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="recentLabel">
         <property name="text">
          <string>Within the last:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="recentSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="3">
        <widget class="QSpinBox" name="aNumberSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2147483647</number>
         </property>
         <property name="value">
          <number>15000</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="6">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="6">
        <spacer name="spacerLeft">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5" rowspan="6">
        <spacer name="spacerRight">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
